core/src/test/java/alt/testcontainers/images/OutOfPackageImagePullPolicyTest.java

```diff
@@ -0,0 +1,27 @@
+package alt.testcontainers.images;
+
+import org.junit.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.startupcheck.OneShotStartupCheckStrategy;
+import org.testcontainers.images.AbstractImagePullPolicy;
+import org.testcontainers.images.ImageData;
+import org.testcontainers.utility.DockerImageName;
+
+public class OutOfPackageImagePullPolicyTest {
+    @Test
+    public void shouldSupportCustomPoliciesOutOfTestContainersPackage() {
+        try (
+            GenericContainer<?> container = new GenericContainer<>("hello-world:latest")
```

Comment:
empty constructor would work too (will use the "default" image), so that we limit the number of images we use in our test suite

Reply:
Fixed. Thanks!